code/__HELPERS/stat_tracking.dm

```diff
@@ -25,3 +27,15 @@
 		)
 
 	rustg_file_write(json_encode(output), "[GLOB.log_directory]/[filename]")
+
+/proc/stat_tracking_export_to_csv_later(filename, costs, counts)
+	if (IsAdminAdvancedProcCall())
+		return
+
+	var/list/output = list()
+
+	output += "key, cost, count"
+	for (var/key in costs)
+		output += "[key], [costs[key]], [counts[key]]"
```

Comment:
You should probably validate (just throwing an exception is fine) that `key` doesn't have any commas

Reply:
gonna just strip all commas, see new pr